src/containers/skills/Skills.css

```diff
@@ -30,7 +30,7 @@
 }
 
 .skills-heading {
-  font-size: 56px;
+  font-size: 52px;
```

Comment:
I think the problem is with `line-height` and not `font-size`

Reply:
Okay then.